services/bettersocial/user/BetterSocialUnblockUserV2.js

```diff
@@ -4,6 +4,16 @@ const {sequelize, User, UserBlockedUser} = require('../../../databases/models');
 const Getstream = require('../../../vendor/getstream');
 const RedisBlockHelper = require('../../redis/helper/RedisBlockHelper');
 const BetterSocialScoreUnblockUser = require('../score/unblock-user');
+
+const isAnonymousUserBlockedFromSignUser = async (selfUserId, anonymousUserId) => {
+  const isBlockedFromSign = await UserBlockedUser.findOne({
+    where: {
+      user_id_blocker: selfUserId,
+      user_id_blocked: anonymousUserId
+    }
+  });
+  return isBlockedFromSign && !isBlockedFromSign.post_id;
+};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `isAnonymousUserBlockedFromSignUser` is missing error handling. If the database query fails for any reason, it will throw an unhandled promise rejection. Consider wrapping the database call in a try-catch block to handle potential errors.
```diff
+  try {
      const isBlockedFromSign = await UserBlockedUser.findOne({
        where: {
          user_id_blocker: selfUserId,
          user_id_blocked: anonymousUserId
        }
      });
      return isBlockedFromSign && !isBlockedFromSign.post_id;
+  } catch (error) {
+    console.error(`Error while checking if anonymous user is blocked: ${error}`);
+    throw error; // or handle error as appropriate
+  }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->